src/site/content/en/progressive-web-apps/using-a-pwa-in-your-android-app/index.md

```diff
@@ -66,8 +66,23 @@ nightly version.
 ### Quality Criteria
 
 Web developers should use a Trusted Web Activity when they want to include web content in an
-Android app. Web content in a Trusted Web Activity must meet Lighthouse's PWA install quality
-criteria and additional Android-specific criteria such as [policy compliance][9].
+Android app.
+
+Web content in a Trusted Web Activity must meet the PWA installability criteria.
+
+Additionally, in order to match the behaviour users expect from Android applications,
+[Chrome 86 introduced a change][18] where failing to handle the following scenarios is considered a
+crash:
+
+  - Failure to verify digital asset links at application launch.
+  - Failure to return HTTP 200 for an offline network resource request.
+  - An HTTP 404 or 5xx error in the application.
```

Comment:
```suggestion
  - An application resource returning an HTTP 404 or 5xx error.
```

Reply:
I feel like I need some extra context here. If I didn't correctly infer what's returning 404 and 5xx, come up with your own wording.

Reply:
How does "A navigation request returning an HTTP 404 or 5xx error" sound?

Reply:
Sounds great.

Reply:
updated!